src/main/resources/db/migration/V0__initial_version.sql

```diff
@@ -0,0 +1,41 @@
+CREATE SCHEMA tasker;
+
+CREATE TABLE tasker.boards
+(
+    id          uuid PRIMARY KEY,
+    title       varchar(50),
+    created_at  timestamptz,
+    description varchar(2000)
+);
```

Comment:
Pls also add `NOT NULL` constraint for fields those not nullable
This comment applies to all table creations